include/oneapi/dpl/pstl/hetero/dpcpp/parallel_backend_sycl.h

```diff
@@ -291,7 +265,8 @@ __parallel_transform_reduce(_ExecutionPolicy&& __exec, _Up __u, _Cp __combine, _
     __work_group_size = oneapi::dpl::__internal::__max_local_allocation_size(::std::forward<_ExecutionPolicy>(__exec),
                                                                              sizeof(_Tp), __work_group_size);
 #if _ONEDPL_COMPILE_KERNEL
```

Comment:
Have you tried to compile/run with `_ONEDPL_COMPILE_KERNEL is 0` as well?

Reply:
Yes but we always set it in 1 because otherwise we have a risk to get runtime crashes and we already met those on CPU